ajax/libs/onecolor/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "onecolor",
+  "filename": "one-color-all.js",
+  "description": "Javascript color object with implicit color space conversions. Supports RGB, HSV, HSL and CMYK with alpha channel.",
+  "license": "BSD-2-clause",
```

Comment:
Since I didn't see this from upstream, @x09326 let's send a PR to contribute upstream having this.